Lenses/Example/src/lens.css

```diff
@@ -0,0 +1,129 @@
+/**
+ * lens.css
+ *
+ * The stylesheet for the Example lens.
+ */
+
+#lens {
+  margin-left: auto;
+  margin-right: auto;
+  padding: 40px 1em;
+  max-width: 1500px;
+}
+
+#loading {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%,-50%);
+}
+
+#nav {
+  text-align: center;
+  border: thin solid;
+  padding: .2em;
+  margin-bottom: 1em;
+}
+
+#nav a {
+  margin: 0 1em;
+  font-size: 1.1em;
+}
+
+.subject, .aspect, .sample {
+  margin: 1em 0;
+  border: thin solid;
+}
+
+.heading {
+  border-bottom: thin solid;
+  text-align: center;
+  padding: 1em;
+  position: relative;
+}
+
+.heading h4 {
+  margin: 0;
+}
+
+.subject .body {
+  display: flex;
+}
+
+.subject .column {
+  flex: 1;
+}
+
+.subject .column.samples {
+  border-left: thin solid;
+}
+
+/* override lightning css */
+ul {
+  margin: 0;
+  padding-left: 1em;
+  list-style-type: disc;
+}
+
```

Comment:
The lightning css interferes with the lens css when loaded into a perspective. Is there a better way to do this? Are there plans to restructure things so this isn't necessary?